run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+declare -a EXTRA_FLAGS=()
+
+# Display Socket
+if [[ $XDG_SESSION_TYPE == "wayland" ]]
+then
+    WAYLAND_SOCKET=${WAYLAND_DISPLAY:-"wayland-0"}
+
+    if [[ -e "$XDG_RUNTIME_DIR/${WAYLAND_SOCKET}" ]]
+    then
+        echo "Wayland socket found, running via Wayland."
+        echo "To run via XWayland, remove the --socket=wayland permission."
+        EXTRA_FLAGS+=(
+            "--enable-features=WaylandWindowDecorations"
+            "--ozone-platform=wayland"
```

Comment:
ozone-platform-hint=auto is the current flag, but imo you shouldn't use any https://github.com/flathub/flathub/pull/4569/files#r1341992103